Formula/bioformats56.rb

```diff
@@ -0,0 +1,32 @@
+class Bioformats < Formula
```

Comment:
To be compatible, the formula should be renamed `Formula/bioformats@5.6.rb` and the class `class BioformatsAT56 < Formula`